meson.build

```diff
@@ -39,6 +39,18 @@ files = []
 includes = []
 has_backends = false
 
+deps += dependency('protobuf', fallback: ['protobuf', 'protobuf_dep'], required: true)
+protoc = find_program('protoc', required: false)
+if not protoc.found()
+  message('protoc will be built from the subproject')
+  protoc = subproject('protobuf').get_variable('protoc')
+endif
+
+gen = generator(protoc, \
+  output    : ['@BASENAME@.pb.cc', '@BASENAME@.pb.h'],
+  arguments : ['--proto_path=@CURRENT_SOURCE_DIR@/libs/lczero-common', '--cpp_out=@BUILD_DIR@', '@INPUT@'])
+
+files += gen.process('libs/lczero-common/lc0net.proto')
```

Comment:
Is `lc0net.proto` located in a root of `lczero-common` repo?
It's better to move it into proto/ subdir, as there may be other things in lczero-common.

Also, maybe it's better to generate it that way so they are also included like:
```cpp
#include "proto/lc0.pb.h"
```
?

I like that more but maybe it's an overkill as we don't expect large tree of protos..
But if you'd like to do that, then:
1. Don't modify --proto-path   (keep it ending at `/libs/lczero-common` rather than `/libs/lczero-common/proto`)
2. in gen.process add parameter:
```
  files += gen.process('libs/lczero-common/proto/lc0net.proto',
  preserve_path_from : meson.current_source_dir() + '/libs/lczero-common/
)

Reply:
Agreed, that would be nicer.